### ISSUE TITLE
Missing quotes from values

### ISSUE THREAD

**Issue by bre7** [vapor/multipart#22]
Missing quotes in values other than encoded file content. Interesting discussion in https://github.com/akka/akka-http/issues/386 (includes spec requirements as well as browser compatibility comments)

E.g:

```HTTP
--BOUNDARY
Content-Disposition: form-data; filename=test.pdf; name=data
Content-Type: application/pdf

%PDF-1.3
...
```

**Comment by tanner0101** [comment]
can you clarify what you mean by this? Maybe an expected vs. actual example.

**Comment by bre7** [comment]
Expected:
```
Content-Disposition: form-data; name="data"; filename="test.pdf"
Content-Type: application/pdf

%PDF-1.3
```

Actual result:

```
--BOUNDARY
Content-Disposition: form-data; filename=test.pdf; name=data
Content-Type: application/pdf

%PDF-1.3
...
```

**Comment by tanner0101** [comment]
Ah, I see. The relevant code is here: https://github.com/vapor/core/blob/master/Sources/Core/HeaderValue.swift#L60-L66

The question is: do we need quotes?

**Comment by Cellane** [comment]
Closing as this was resolved in vapor/core#144 and vapor/vapor#1668. Thank you so much!
